dw-checkbox.js

```diff
@@ -136,6 +136,7 @@ export class DwCheckbox extends DwFormElement(LitElement) {
   _onChange(e) { 
     this.checked = e.target.checked;
     this.dispatchEvent(new Event('checked-changed', e));
```

Comment:
As `checked-changed` event is being deprecated. It's documentation should be updated to state the same.